doc/mgmt/SONiC_IntfRangeFeature_HLD.md

```diff
@@ -0,0 +1,235 @@
+# Interface Range Specification support
+Implement support for Interface range Specification via KLISH CLI in SONiC management framework.
+
+# High Level Design Document
+#### Rev 0.1
+
+# Table of Contents
+* [List of Tables](#list-of-tables)
+* [Revision](#revision)
+* [About This Manual](#about-this-manual)
+* [Scope](#scope)
+* [Definition/Abbreviation](#definitionabbreviation)
+
+# List of Tables
+[Table 1: Abbreviations](#table-1-abbreviations)
+# Revision
+| Rev | Date | Author | Change Description |
+|:---:|:----------:|:-----------------------:|----------------------------------------------|
+| 0.1 | 05/17/2020 | Tejaswi Goel | Initial version |
+
+# Scope
+* Implementation details, syntax, and unit test cases regarding interface range command support via KLISH CLI.
+* Interface range specification will not be supported via REST/gNMI.
+* Breakout command will not be supported in interface range configuration mode.
+
+# Definition/Abbreviation
+NA
+
+# 1 Feature Overview
+
+Interface Range Specification feature support via Klish CLI will allow specification of range of interfaces to which the subsequent commands will be applied. Configuring a range of interfaces will help reduce the time and effort in configuring interfaces.
```

Comment:
How much time reduction will be there, if large number of vlan interfaces say 1-4000 are created or deleted, compared to the same config when using single command